PC/winreg.c

```diff
@@ -1394,7 +1411,7 @@ winreg.OpenKey -> HKEY
     sub_key: Py_UNICODE(accept={str, NoneType})
         A string that identifies the sub_key to open.
     reserved: int = 0
-        A reserved integer that must be zero.  Default is zero.
+        A reserved integer that must be zero. Default is zero.
```

Comment:
This isn't really true anymore - it's now passed to `options`.

It's not immediately obvious to me whether we should add `options` here as well like for OpenKeyEx, or just require that it be zero. Maybe pass it as `reserved` so there's a warning, and then make it an exception in 3.16?

Reply:
My main concern is someone passing a value here and seeing different behaviour and then having to figure out why. Since we don't say that the value is actually used for anything, it just makes it harder to figure out why it's having an effect.

Reply:
Updated to pass it to `winreg.CreateKeyEx` with parameter `options`, thus it will warn a `DeprecatedWarning`, and updated the document to remove it in `3.16`.